setup.py

```diff
@@ -40,18 +39,15 @@ def main():
             'Natural Language :: English',
             'Operating System :: OS Independent',
             'Programming Language :: Python :: 2.7',
-            'Programming Language :: Python :: 3.5',
             'Programming Language :: Python :: 3.6',
+            'Programming Language :: Python :: 3.7',
             'Programming Language :: Python',
             'Topic :: Database',
```

Comment:
Remove `Topic :: Database` and check if we should update `Development Status :: 4 - Beta`

Reply:
Also update `url="..."`

Reply:
After looking at all of the options for `Development Status` I think we should leave it as it is.